includes/class-wc-cart-session.php

```diff
@@ -239,4 +236,125 @@ private function set_cart_cookies( $set = true ) {
 		}
 		do_action( 'woocommerce_set_cart_cookies', $set );
 	}
+
+	/**
+	 * Get the persistent cart from the database.
+	 *
+	 * @access private
```

Comment:
No need to use `@access` since the method is already private, we should remove all `@access` tags inside classes.